src/test/shell/bazel/remote/remote_execution_http_test.sh

```diff
@@ -341,4 +341,72 @@ EOF
     || fail "Failed to build //a:gen1 without remote cache"
 }
 
+function test_cc_combined_disk_http_cache() {
+  # Test for the combined disk and http cache.
+  # Built items should be pushed to both the disk and http cache.
+  # If an item is missing on disk cache, but present on http cache,
+  # then bazel should copy it from http cache to disk cache on fetch.
+
+  local cache="${TEST_TMPDIR}/cache"
+  local disk_flags="--disk_cache=$cache"
+  local http_flags="--remote_http_cache=http://localhost:${http_port}"
+
+  mkdir -p a
+  cat > a/BUILD <<EOF
+package(default_visibility = ["//visibility:public"])
+cc_binary(
```

Comment:
I know we are using `cc_binary` through the remote execution tests, but we are currently working towards removing it completely where not necessary and using a faster `genrule` (i.e one that writes hello world to some file) instead in order to speed up our tests.

Reply:
Makes sense. I've changed the test case to use a `genrule` instead.